### PR TITLE
BEdigitech Bid Adapter: New Bid Adapter 

### PR DESCRIPTION
**Pull request by beglobal2022**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by beglobal2022** [comment]
@jsnellbaker Document PR: https://github.com/prebid/prebid.github.io/pull/4394

**Review by lksharma — CHANGES_REQUESTED** [review]

File: modules/bedigitechBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  code: BEDIGITECH_CODE,
+  supportedMediaTypes: [BANNER, NATIVE],
+  isBidRequestValid: bid => {
+    return !!bid.params.placementId && !!bid.bidId && bid.bidder === 'bedigitech'
+  },
+
+  buildRequests: (bidRequests, bidderRequest) => {
+    return bidRequests.map(bid => {
+      let url = BEDIGITECH_ENDPOINT + '?pid=' + bid.params.placementId;
+      const data = {};
```
[the remark this answers is not in the document]
Comment (beglobal2022) [reply modules/bedigitechBidAdapter.js]: Done

[the remark this answers is not in the document]
Comment (beglobal2022) [reply modules/bedigitechBidAdapter.js]: > Query params should be part of the `data` object.

@lksharma This has been fixed.

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**